packages/web-react/src/components/Alert/index.demo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import DocsSection from '../../../docs/DocsSections';
+import AlertCentered from './stories/AlertCentered';
+import AlertColors from './stories/AlertColors';
+import AlertIcons from './stories/AlertIcons';
+
+ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
```

Comment:
Could we also leave this file in `stories/` ?

Reply:
Why? This is not a story. But I can leave it in `/demo/index.ts` directory. Anyway, I think it is the same. We can use `/demo/index.tsx` or `index.demo.tsx` but it will be in every component.